python/dgl/nn/pytorch/conv/cfconv.py

```diff
@@ -123,18 +123,21 @@ def forward(self, g, node_feats, edge_feats):
         ----------
         g : DGLGraph
             The graph.
-        node_feats : float32 tensor of shape (V, node_in_feats)
-            Input node features, V for the number of nodes.
-        edge_feats : float32 tensor of shape (E, edge_in_feats)
-            Input edge features, E for the number of edges.
+        node_feats : torch.Tensor
+            The input node feature of shape :math:`(N_{in}, node_in_feats)`
+            where :math:`N_{in}` is the number of source nodes.
+        edge_feats : torch.Tensor
+            The input edge feature of shape :math:`(E, edge_in_feats)`
+            where :math:`E` is the number of edges.
 
         Returns
         -------
-        float32 tensor of shape (V, out_feats)
-            Updated node representations.
+        torch.Tensor
+            The output node feature of shape :math:`(N_{out}, out_feats)`
+            where :math:`N_{out}` is the number of destination nodes.
         """
         with g.local_scope():
-            g.ndata['hv'] = self.project_node(node_feats)
+            g.srcdata['hv'] = self.project_node(node_feats)
```

Comment:
For using with HeteroGraphConv a module must support accepting a pair of source node features and destination node features.  You will need to do something like:
```python
if isinstance(node_feats, tuple):
    node_feats_src, _ = node_feats
else:
    node_feats_src = node_feats
g.srcdata['hv'] = self.project_node(node_feats_src)
# ...
```